staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -108,6 +108,11 @@ const (
 	// Enables expression validation in Admission Control
 	ValidatingAdmissionPolicy featuregate.Feature = "ValidatingAdmissionPolicy"
 
+	// owner: @serathius
+	// beta: 1.30
+	// Enables consistent watches to be served from etcd directly.
+	ConsistentWatchFromEtcd featuregate.Feature = "ConsistentWatchFromEtcd"
```

Comment:
Wanted to follow the same naming convention as `ConsistentListFromCache` (in alpha, planned for beta in 1.31), and `ConsistentWatchFromCache` planned for 1.31.

It's called consistent watch/list because it tries to get a newest state, a read that is consistent with the state of all members of the cluster. It's not related to load.

What about `ConsistentWatchFromStorage`?

Reply:
Changed to `WatchFromCacheWithoutResourceVersion`. PTAL